dspot-maven/src/main/java/eu/stamp_project/DSpotMojo.java

```diff
@@ -19,132 +22,146 @@
     private static final Logger LOGGER = LoggerFactory.getLogger(DSpotMojo.class);
 
     /**
-     * [mandatory] specify the path to the configuration file (format Java properties) of the target project (e.g. ./foo.properties).
+     *	[mandatory] specify the path to the configuration file (format Java properties) of the target project (e.g. ./foo.properties).
      */
-    @Parameter(property = "pathToPropertiesFile", required = true)
-    private String pathToPropertiesFile;
+    @Parameter(property = "path-to-properties")
+    private String pathToProperties;
 
     /**
-     * [optional] specify the list of amplifiers to use. Default with all available amplifiers.
-     * - StringLiteralAmplifier
-     * - NumberLiteralAmplifier
-     * - CharLiteralAmplifier
-     * - BooleanLiteralAmplifier
-     * - AllLiteralAmplifiers
-     * - MethodAdd
-     * - MethodRemove
-     * - TestDataMutator (deprecated)
-     * - StatementAdd
-     * - ReplacementAmplifier
-     * - None
+     *	[optional] specify the list of amplifiers to use. Default with all available amplifiers.
```

Comment:
This doesn't seem the case. it says "None" below as the default value for `amplifiers` :)